app/views/reports/elements/_step_element.html.erb

```diff
@@ -27,9 +27,10 @@
     <div class="row">
       <div class="col-xs-12 ql-editor">
         <% if strip_tags(step.description).present? %>
-          <%= auto_link(step.description,
-                        link: :urls,
-                        html: { target: '_blank' }).html_safe %>
+          <%= auto_link(smart_annotation_parser(step.description),
```

Comment:
Forgot the simple_format. Actually, I suggest to embed simple_format call into the smart_annotation_parser, as it should be used everytime.